recipe/meta.yaml

```diff
@@ -24,16 +27,20 @@ requirements:
     - curl  # [win]
     - jom  # [win]
     - objconv  # [osx]
-    - llvm-openmp  # [osx or linux]
+    - llvm-openmp  # [(osx or linux) and USE_OPENMP == 1]
     # Following is needed because the Fortran compiler add an rpath to the BUILD_PREFIX
-    - _openmp_mutex * *_llvm  # [linux]
+    - _openmp_mutex * *_llvm  # [linux and USE_OPENMP == 1]
   host:
   run:
 
 outputs:
   - name: libopenblas{{ name_suffix }}
     build:
       skip: true  # [win and vc<14]
+      string: openmp_h{{ PKG_HASH }}_{{ build_number }}         # [USE_OPENMP == 1]
+      string: pthreads_h{{ PKG_HASH }}_{{ build_number }}       # [USE_OPENMP == 0]
+      track_features:
+        - openblas_threading  # [linux and USE_OPENMP == 1]
```

Comment:
Do we want this feature name to be more specific to openmp? 

This might help ppl understand envs later if there are issues. 

NBD either way TBH.

Reply:
```suggestion
        - openblas_threading_openmp  # [linux and USE_OPENMP == 1]
```